utils/vlog/yosys/json.py

```diff
@@ -122,6 +162,38 @@ def cell_conns(self, cell, direction="input"):
                         conns.append(("{}[{}]".format(port, i), condata[i]))
         return conns
 
+    def cell_conn_list(self, cell, port):
+        """The connections of a cell in a given port as a list
+
+        Returns a list of connections:
+        -------
+        net : list
```

Comment:
Please follow commenting style from https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings

Reply:
I'm pretty sure we have be using the [numpy style for docstring comments](https://numpydoc.readthedocs.io/en/latest/format.html)?

Reply:
> I'm pretty sure we have be using the [numpy style for docstring comments](https://numpydoc.readthedocs.io/en/latest/format.html)?

I'm actually happy to see either.   I think picking a convention (and writing it down) is a good plan.

FYI looks like https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html supports both conventions.  They list the following note:

```
NumPy style tends to require more vertical space, whereas Google style tends to use more horizontal space. Google style tends to be easier to read for short and simple docstrings, whereas NumPy style tends be easier to read for long and in-depth docstrings.

The Khan Academy recommends using Google style.
```

I will say that prjxray generally is missing a significant number of docstrings, so it cannot be a guide.  symbiflow-arch-defs currently has a mix of styles, but given that we haven't been enforcing docstring style, it is unclear if either convention is actually being used.  I suggest we consider adding a docstring lint pass.